src/zabapgit_objects.prog.abap

```diff
@@ -987,14 +987,20 @@ CLASS lcl_objects_program IMPLEMENTATION.
           lt_source       TYPE TABLE OF abaptxt255,
           lt_tpool        TYPE textpool_table,
           ls_tpool        LIKE LINE OF lt_tpool,
-          lo_xml          TYPE REF TO lcl_xml_output.
+          lo_xml          TYPE REF TO lcl_xml_output,
+          lv_subrc        LIKE sy-subrc,
+          lv_curr_lang    TYPE langu,
```

Comment:
This could be renamed to lv_current_language for clarity

Reply:
I agree.